spec/factories/rankinglists.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :rankinglist, parent: :event, class: :rankinglist do
+    game_mode Rankinglist.game_modes[Rankinglist.game_modes.keys.sample]
+    initial_value 1.3
+    deadline nil
+    startdate nil
+    enddate nil
+    player_type nil
```

Comment:
This is definitely not the cleanest solution to a good rankinglist factory. In my opinion. the :event factory should not specify all the dates and player_type (if you end up actually not using it), but instead have a subfactory `:event_wtih_dates`
that specifies all these values. That way you can inherit from the base factory and can cut out the lines 5-8. To make tournaments and leagues work again they just have to inherit from `:event_with_dates.`